Update tokenizer to only process at-mentions

eruizdechavez opened the pull request:
Resolves #21

Commit:
Update tokenizer to only process at-mentions

Resolves #21

IngRivera91 reviewed (changes requested):
quitar el else de la linea 36 y poner su propi if ($token === '--' )

Commit:
Fix an issue introoduced on the previuous commit

Any non mention token would enter to the decrease if/else and reset
the buffer content.

Commit:
Reformat code

Validated and fixed code against PSR12 using phpcs and phpcbf.

eruizdechavez commented:
I linted the code inside src in my last commits against PSR12 with phpcs and phpcbf. To make it easier to review, you can use the “Hide whitespace changes” setting in the “Files changed” tab.

eclipxe13 committed:
Refactor and include tests (#2)

* Add tokenizer test expectations

* Add mix and duplicates cases

* Refactor tokenizer and pass tests

* Tokenizer should not deduple or balance
